diff --git a/arch/arm/boot/dts/am33xx.dtsi b/arch/arm/boot/dts/am33xx.dtsi
index a89d5e2cf01b..7ab532a05be2 100644
--- a/arch/arm/boot/dts/am33xx.dtsi
+++ b/arch/arm/boot/dts/am33xx.dtsi
@@ -464,7 +464,11 @@
 				"NC",	// 30
 				"NC";	// 31
 		};
-
+		my_bus: my_bus@44e19000 {
+			compatible = "my_bus";
+			reg = <0x44e19000 0x1000>;
+			status = "okay";
+		};
 		uart0: serial@44e09000 {
 			compatible = "ti,am3352-uart", "ti,omap3-uart";
 			ti,hwmods = "uart1";
